symengine/dense_matrix.cpp

```diff
@@ -1613,6 +1631,25 @@ void cross(const DenseMatrix &A, const DenseMatrix &B, DenseMatrix &C)
 
 RCP<const Set> eigen_values(const DenseMatrix &A)
 {
+    unsigned n = A.nrows();
+    bool is_upper = true, is_lower = true;
+    for (unsigned i = 0; i < n; ++i) {
+        for (unsigned j = 0; j < n; ++j) {
+            if (i > j and not eq(*(A.get(i, j)), *zero))
```

Comment:
Can you use braces?
```suggestion
            if (i > j and not eq(*(A.get(i, j)), *zero)) {
```

Reply:
Okay I am sending a PR right now. Thanks for reviewing :)